README.md

```diff
@@ -1,7 +1,7 @@
 AnitaB.org Portal - VMS project
 ============================
 
-[![Build Status](https://travis-ci.org/anitab-org/vms.svg?branch=develop)](https://travis-ci.org/anitab-org/vms) [![Coverage Status](https://coveralls.io/repos/github/anitab-org/vms/badge.svg?branch=develop)](https://coveralls.io/github/anitab-org/vms?branch=develop) [![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg?style=flat-square)](http://makeapullrequest.com) [![project chat](https://img.shields.io/badge/zulip-join_chat-brightgreen.svg)](https://anitab-org.zulipchat.com/#narrow/stream/222539-vms)
+[![Build Status](https://travis-ci.org/anitab-org/vms.svg?branch=develop)](https://travis-ci.org/anitab-org/vms) [![Coverage Status](https://coveralls.io/repos/github/anitab-org/vms/badge.svg?branch=develop)](https://coveralls.io/github/anitab-org/vms?branch=develop) [![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg?style=flat-square)](http://makeapullrequest.com) [![project chat](https://img.shields.io/badge/zulip-join_chat-brightgreen.svg)](https://anitab-org.zulipchat.com/#narrow/stream/222539-vms) ![Build Status](https://codecov.io/gh/anitab-org/portal/branch/develop/graph/badge.svg)
```

Comment:
Here, add codecov for the VMS, not for the portal, also add the codecov link to it.

Reply:
Add some more description for the PR.

Reply:
Thanks for the correction, I will add the codecov Link to it as well.

But regarding the 'not adding it to portal' comment, I have appended the codecov to readme.md file of `vms` project exactly, and not elsewhere.

Reply:
No problem, you can update the codecov link. 

Yes right, can you update this `AnitaB.org Portal - VMS project` to `AnitaB.org - Volunteer Management System(VMS) project`.

Reply:
@satya7289 just a tiny concern here regarding the project title.
I observed the readme-files of other projects and most of them have project names itself in the readme file, for instance,
`Mentorship System (Backend)` and not `AnitaB.org - Mentorship System (Backend)`
`Mentorship System (Android)`
`Mentorship System (iOS)`
and so on...

So should we keep it in uniform with others which will impart more uniformity? 
If yes then it will be as follows
`Volunteer Management System(VMS)`  instead of AnitaB.org - Volunteer Management System(VMS) project
Please let me know. Thanks.